src/toolchain.ml

```diff
@@ -0,0 +1,260 @@
+open Import
+
+let opamroot =
+  let home = Node.Os.homedir () in
```

Comment:
I changed this to use Node's `os.homedir` because Windows doesn't use the `$HOME` environmental variables.

Before this change, the extension failed to activate on Windows.

Reply:
I was sure Windows used `HOME`! Thanks @mnxn